### ISSUE TITLE
Exporting skeletons as object files

### ISSUE THREAD

**Issue by conochur** [navis-org/pymaid#243]
I know that there's a feature in the GUI to export skeleton selections as .obj files, but can one do the same using PyMaid?

**Comment by schlegelp** [comment]
Yes, in theory! In practice it depends a bit on what exactly it is you are after.

The `obj` format is (to my knowledge) most commonly used for meshes, i.e. an object with faces and vertices. It's pretty straight forward to convert a skeleton into a tubular mesh. `.obj` appears to also support curve/line objects though. Not sure which of the two option CATMAID is exporting?

**Comment by conochur** [comment]
Thanks for the response! I'm looking for a mesh file that I can pop into other rendering software. But would like to do it programmatically. Do you have any recommendations for accomplishing that from python using the coordinate and radius data?
